views/components/header.ejs

```diff
@@ -39,4 +39,4 @@
                 </ul>
             </div>
         </div>
-    </nav>
+    </nav>
```

Comment:
You can just quick fix the newline issues here

Reply:
Added a new line merge this PR